android/pom.xml

```diff
@@ -322,7 +321,7 @@
             <runOrder>alphabetical</runOrder>
             <!-- Set max heap for tests. -->
             <!-- Catch dependencies on the default locale by setting it to hi-IN. -->
-            <argLine>-Xmx1536M -Duser.language=hi -Duser.country=IN ${test.add.args} ${test.add.opens}</argLine>
```

Comment:
is the `test.add.args` field used? It appears to be always empty.

https://github.com/search?q=repo%3Agoogle%2Fguava%20test.add.args&type=code

Reply:
SOC